### DISCUSSION TITLE
FullStack (Customizar o Register)

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#670] [Tira Dúvidas] [answered]
No nosso curso de Fullstach na classe user eu coloquei um campo a mais chamado id_empresa, esse campo já associado a tabelas do identity nos migrations, como faço para que no register receba esse campo também?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @FBODeveloper, como vai?

Se você usar o método padrão do Identity, não tem como setar este valor, você precisa customizar o método, por este motivo normalmente separamos o usuário da aplicação do cliente.

Sabe o método "logout" que criei no curso? Você precisaria fazer algo naquela linha, mas para o "register", substituindo o comportamento padrão do Identity.

**Reply by FBODeveloper** [reply]
Entendi... então tenho que criar um Endpoint novo pra isso...
Valeu Balta.
